vite.config.ts

```diff
@@ -4,5 +4,7 @@ import react from '@vitejs/plugin-react'
 // https://vitejs.dev/config/
 export default defineConfig({
   plugins: [react()],
-
-})
+  server: {
+    open: true,
```

Comment:
Strange that this isn't needed in the OSS project to achieve the same behaviour, perhaps its due to this project using a newer version of vite...

Reply:
Perhaps